module/localisation/RobotLocalisation/src/RobotLocalisation.hpp

```diff
@@ -42,6 +42,8 @@
 namespace module::localisation {
     class RobotLocalisation : public NUClear::Reactor {
     private:
+        uint PLAYER_ID;
```

Comment:
```suggestion
        uint PLAYER_ID = 1;
```
technically its not a constant and shouldn't be upper case

Reply:
yep